languages/English/English_global.php

```diff
@@ -2,10 +2,291 @@
 	/**
 	 * e107 website system
 	 *
-	 * Copyright (C) 2008-2016 e107 Inc (e107.org)
+	 * Copyright (C) 2008-2021 e107 Inc (e107.org)
 	 * Released under the terms and conditions of the
 	 * GNU General Public License (http://www.gnu.org/licenses/gpl.txt)
 	 *
 	 */
 
-define("LAN_PLUGIN_VSTORE_NAME", "Shop");
\ No newline at end of file
+define("LAN_PLUGIN_VSTORE_NAME", "Shop");
+define("LAN_PLUGIN_VSTORE_PRICE", "Price");
+
```

Comment:
English_global.php should remain unchanged. Everything should be in English_front.php and English_admin.php